concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-168	168	to torture, to make suffer	1952	TORTURE
+RonaTas-2011-431-169	169	to conquer (somebody, something), triumph (over somebody, something), gain victory	782	DEFEAT
+RonaTas-2011-431-170	170	to catch fire, be kindled, to be ignited		
+RonaTas-2011-431-171	171	to knead, pug	274	KNEAD
+RonaTas-2011-431-172	172	to assemble, gather	3638	ASSEMBLE (GET TOGETHER)
+RonaTas-2011-431-173	173	to collect,gather something	7	GATHER
+RonaTas-2011-431-174	174	to hate	1607	HATE
+RonaTas-2011-431-175	175	fruit	1507	FRUIT
+RonaTas-2011-431-176	176	hawthorn, a plant similar to the whitethorn, or elder	1168	HAWTHORN
+RonaTas-2011-431-177	177	a kind of tree similar to the maple or cornel, Acer tataricum, Cornussanguinea		
+RonaTas-2011-431-178	178	ring	1374	RING
+RonaTas-2011-431-179	179	thimble		
+RonaTas-2011-431-180	180	boat, ship	1844	BOAT
+RonaTas-2011-431-181	181	bell	2246	BELL
+RonaTas-2011-431-182	182	hawk	307	HAWK
+RonaTas-2011-431-183	183	sand	671	SAND
+RonaTas-2011-431-184	184	noose, loop, snare	2604	NOOSE
+RonaTas-2011-431-185	185	time, weather	892	TIME
+RonaTas-2011-431-186	186	word, biblical word, verb
+RonaTas-2011-431-187	187	yes; affirmative particle, very	1268	YES
+RonaTas-2011-431-188	188	to be frightened, to take fright		
+RonaTas-2011-431-189	189	twin	1005	TWINS
+RonaTas-2011-431-190	190	proper behaviour		
+RonaTas-2011-431-191	191	to suit something, to be proper, to fit into		
+RonaTas-2011-431-192	192	to adore, worship	257	WORSHIP
+RonaTas-2011-431-193	193	shirt	1622	SHIRT
+RonaTas-2011-431-194	194	to write	1672	WRITE
+RonaTas-2011-431-195	195	buttermilk		
+RonaTas-2011-431-196	196	to glow, be hot	1286	HOT
+RonaTas-2011-431-197	197	name of an ethnic group of Iranian origin in Hungary		
+RonaTas-2011-431-198	198	do-less, do-little		
+RonaTas-2011-431-199	199	bulrush, club-rush, different plants like Schoenoplectus (Cyperaceæ)		
+RonaTas-2011-431-200	200	guide, pilot; conductor, ticket inspector	43	LEAD (GUIDE)
+RonaTas-2011-431-201	201	an ethnic group of Muslim faith in Mediaeval Hungary		
+RonaTas-2011-431-202	202	a water plant, Stratiotes aloides		
+RonaTas-2011-431-203	203	whip, lash	2630	WHIP
+RonaTas-2011-431-204	204	saddle strap		
+RonaTas-2011-431-205	205	bridle, reins		
+RonaTas-2011-431-206	206	measles	3389	MEASLES
+RonaTas-2011-431-207	207	trap, snare	1075	TRAP (PITFALL)
+RonaTas-2011-431-208	208	gate	2940	GATE
+RonaTas-2011-431-209	209	arm	1673	ARM
+RonaTas-2011-431-210	210	damage, loss	2947	DAMAGE (INJURY)
+RonaTas-2011-431-211	211	man of grit, stout fellow		
+RonaTas-2011-431-212	212	sheepfold, cattlegrid, stockyard, a yard encircled by a fence, shed for animals		
+RonaTas-2011-431-213	213	pink, several kinds of pink, Danthus, D. caryophyllus, several members of the Caryophillaceæ
+RonaTas-2011-431-214	214	stake, pale, stick	1295	STICK
+RonaTas-2011-431-215	215	a fishing water bird with black feathers, cormorant,Phalacrocorax carbo	840	CORMORANT
+RonaTas-2011-431-216	216	sparrow hawk, Accipiter n. nisus		
```

Comment:
Maybe this could be mapped to 307 HAWK. This is the German "Sperber" so it's not exactly a hawk, but they belong to the same family and even the same genus (Accipiter).

Reply:
see issue about again